wordpress-mobile/WordPress-Android
# Changing switch site button height to wrap_content to adapt to system font size change

Fixes #11839 

As per @khaykov comment [here](https://github.com/wordpress-mobile/WordPress-Android/issues/11839#issuecomment-632421585) , changing the height of the 'SWITCH SITE' botton to `wrap_content` so to account for user possibly setting Largest font size in system `Settings > Display > Advanced > Font size`

Since switching to wrap content make the inner padding of the button to come into place I have reduced accordingly the top/bottom margins to get the below effect that to me seems to work well enough (in the images on the left we have the current develop version and on the right this PR patched version).

| Font Size Small | ![image](https://user-images.githubusercontent.com/47797566/82713407-301fd080-9c8b-11ea-8e29-72e77e1f653e.png) |
|---|---|
| Font Size Default | ![image](https://user-images.githubusercontent.com/47797566/82713435-43cb3700-9c8b-11ea-9075-502c75b7cbd3.png) |
|Font Size Large | ![image](https://user-images.githubusercontent.com/47797566/82713456-53e31680-9c8b-11ea-94f2-aacf5a9cf3c1.png) |
| Font Size Largest | ![image](https://user-images.githubusercontent.com/47797566/82713482-61000580-9c8b-11ea-9050-523a80127234.png) |

## To test
- with current develop check you get the Switch Site text cut with Largest font
- with this PR version check you get the switch site text not cut with Largest font
- check that with the other possible font size you get a correctly visualized switch site button consistent with above images.

PR submission checklist:
